src/integration/spring/src/jvmMain/kotlin/community/flock/wirespec/integration/spring/java/Extensions.kt

```diff
@@ -0,0 +1,38 @@
+package community.flock.wirespec.integration.spring.java
+
+import community.flock.wirespec.java.Wirespec
+import java.io.BufferedReader
+import java.lang.reflect.Method
+import java.util.function.Function
+import kotlin.reflect.full.companionObjectInstance
+
+fun Class<*>.getStaticClass(): Class<*> =
+    if (isKotlinClass()) {
+        kotlin.companionObjectInstance?.javaClass ?: error("not found")
+    } else {
+        this
+    }
+
+fun Class<*>.getStaticMethod(name: String): Method? = getStaticClass().methods.find { it.name == name }
```

Comment:
IntelliJ suggests this method is not used. Removing this reveals the method at line 9 is subsequently also not used. Can they be removed?

Reply:
removed